src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/SessionCompletionState.java

```diff
@@ -22,8 +22,8 @@
     DECLINED;
 
     // These include all sessions except sessions that are not yet available, and thus cannot be included
-    // in a calculation of adherence. 
+    // in a calculation of adherence.
     public static final EnumSet<SessionCompletionState> NONCOMPLIANT = EnumSet.of(ABANDONED, EXPIRED, DECLINED);
     public static final EnumSet<SessionCompletionState> COMPLIANT = EnumSet.of(COMPLETED);
-    public static final EnumSet<SessionCompletionState> UNKNOWN = EnumSet.of(UNSTARTED, STARTED);
+    public static final EnumSet<SessionCompletionState> UNKNOWN = EnumSet.of(UNSTARTED, STARTED, NOT_YET_AVAILABLE);
```

Comment:
So this makes sense for *weekly* adherence, but what about *study-scoped* adherence?

Reply:
0-100 searches of weekly adherence reports would not return records that have NULL, so don't test >0 and <100.

Reply:
0-100 searches of weekly adherence reports would not return records that have NULL, so don't test >0 and <100.

Reply:
Yes according to design, in the week and for the study as a whole, adherence starts at 0% and works its way up to 100%.

Reply:
On the second, code is adjusted so that adherence range search does not search on this value if the two fields are null, searches on a range unbounded in one direction if one is specified, or both if both are specified. If a record has no min/max values and min/max values are specified in the search, the record is not returned.